corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -3,6 +3,7 @@ hqDefine(
     [
         "jquery",
         "knockout",
+        "underscore",
```

Comment:
You also need to add `_` as the third parameter in the function call: each dependency is matched up with the corresponding parameter, which is then used in the function body to access that dependency. Dependencies that aren't directly referenced don't need a parameter.

I'd expect this code to fail when it references `initialPageData`, which is correspond to underscore now instead of the initial page data module.

Reply:
Thanks. Added _ as third param